pcsx2/gui/MainMenuClicks.cpp

```diff
@@ -893,18 +908,12 @@ void MainEmuFrame::Menu_Recording_Stop_Click(wxCommandEvent &event)
 
 void MainEmuFrame::Menu_Recording_VirtualPad_Open_Click(wxCommandEvent &event)
 {
-	VirtualPad *vp = NULL;
-	if (event.GetId() == MenuId_Recording_VirtualPad_Port0)
-	{
-		vp = wxGetApp().GetVirtualPadPtr(0);
-	}
-	else if (event.GetId() == MenuId_Recording_VirtualPad_Port1)
-	{
-		vp = wxGetApp().GetVirtualPadPtr(1);
-	}
-	if (vp != NULL)
+	switch (event.GetId() - MenuId_Recording_VirtualPad_Port0)
 	{
-		vp->Show();
+	case 0:
+		wxGetApp().GetVirtualPadPtr(0)->Show(); break;
+	case 1:
+		wxGetApp().GetVirtualPadPtr(1)->Show(); break;
```

Comment:
just pass the result from `event.GetId() - MenuId_Recording_VirtualPad_Port0` into `GetVirtualPadPtr`.  Handling out of range values can be done in that function if needed.

Reply:
Ooooo, was thinking this one over and that's actually not a bad idea. Could go well with multitap too with some bitwise work.